xsdata/parser.py

```diff
@@ -109,9 +110,11 @@ def set_schema_namespaces(self, obj: xsd.Schema, element: etree.Element):
         namespaces if the are missing xsi, xlink, xml, xs."""
         obj.target_namespace = obj.target_namespace or self.target_namespace
 
+        self.set_namespace_map(element, obj)
+
+    def set_namespace_map(self, element, obj):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5141157)